### PR TITLE
Changes references to docker/docker to moby/moby

### LINKED ISSUE #2868: Change references to docker/docker to moby/moby
### Problem description

Hitting `docker/docker` now redirects to `moby/moby` but we should clean up mentions of it in:
- docs
- Dockerfile and `_scripts/` files
- `_layouts/docs.html` and associated Javascript
- `_data/not_edited_here.yaml`

### PR DESCRIPTION
**Pull request by shubheksha**
### Related issues (optional)
Fixes #2868

### CODE REVIEW

File: cs-engine/1.12/release-notes/release-notes.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ CS Engine. For notes on older versions, see the [CS Engine prior release notes a
 (28 Feb 2017)
 
 * Fixes an issue causing containerd to deadlock [#336](https://github.com/docker/containerd/pull/336)
-* Fixes an issue where encrypted overlay networks stop working [#30727](https://github.com/docker/docker/issues/30727)
+* Fixes an issue where encrypted overlay networks stop working [#30727](https://github.com/moby/moby/issues/30727)
```
Comment (mstanleyjones) [inline cs-engine/1.12/release-notes/release-notes.md]: Please revert all the  changes in this file.

File: cs-engine/1.12/release-notes/prior-release-notes.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ title: Release notes archive for Commercially Supported Docker Engine.
 This document contains the previous versions of the commercially supported
 Docker Engine release notes. It includes issues, fixes, and new features.
 
-Refer to the [detailed list](https://github.com/docker/docker/releases) of all changes since the release of CS Engine 1.10.3-cs3
+Refer to the [detailed list](https://github.com/moby/moby/releases) of all changes since the release of CS Engine 1.10.3-cs3
```
Comment (mstanleyjones) [inline cs-engine/1.12/release-notes/prior-release-notes.md]: Please revert all the changes in this file.

File: cs-engine/1.13/release-notes.md [hunk 3]
```diff
@@ -30,35 +30,35 @@ Backports all fixes from 17.03.1
 
 ### Client
 
-* Fix panic in `docker stats --format` [#30776](https://github.com/docker/docker/pull/30776)
+* Fix panic in `docker stats --format` [#30776](https://github.com/moby/moby/pull/30776)
```
Comment (mstanleyjones) [inline cs-engine/1.13/release-notes.md]: Please revert all the changes in this file.

File: docker-for-aws/release-notes.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ Release date: 03/30/2017
 
 **New**
 
-- Docker Engine upgraded to [Docker 17.03.1 CE](https://github.com/docker/docker/blob/master/CHANGELOG.md)
+- Docker Engine upgraded to [Docker 17.03.1 CE](https://github.com/moby/moby/blob/master/CHANGELOG.md)
```
Comment (mstanleyjones) [inline docker-for-aws/release-notes.md]: Please revert the changes in this file.

File: docker-for-azure/release-notes.md [hunk 5]
```diff
@@ -16,7 +16,7 @@ Release date: 03/30/2017
 
 **New**
 
-- Docker Engine upgraded to [Docker 17.03.1 CE](https://github.com/docker/docker/blob/master/CHANGELOG.md)
+- Docker Engine upgraded to [Docker 17.03.1 CE](https://github.com/moby/moby/blob/master/CHANGELOG.md)
```
Comment (mstanleyjones) [inline docker-for-azure/release-notes.md]: Please revert all the changes in this file.

File: engine/api/get-started.md [hunk 6]
```diff
@@ -66,7 +66,7 @@ You can specify the API version to use, in one of the following ways:
 
 - For the SDKs, you can also specify the API version programmatically, as a
   parameter to the `client` object. See the
-  [Go constructor](https://github.com/docker/docker/blob/master/client/client.go#L136){: target="_blank" class="_"}
+  [Go constructor](https://github.com/moby/moby/blob/master/client/client.go#L136){: target="_blank" class="_"}
```
Comment (mstanleyjones) [inline engine/api/get-started.md]: I think the changes in the **code** in this file need to be reverted.

File: 404.md [hunk 7]
```diff
@@ -68,7 +68,7 @@ if (gonnaFwd) {
   console.log("Showing 404 choices");
   var ghIssueQueryString = "title=404%20at%20" + path + "&body=URL%3A%20" + path;
   var output = new Array();
-  output.push("<ul><li><a href='https://github.com/docker/docker.github.io/issues/new?" + ghIssueQueryString + "' class='nomunge'>File a ticket</a></li>");
+  output.push("<ul><li><a href='https://github.com/moby/moby.github.io/issues/new?" + ghIssueQueryString + "' class='nomunge'>File a ticket</a></li>");
```
Comment (mstanleyjones) [inline 404.md]: This is definitely wrong.

File: engine/api/get-started.md (hunk 6 above)
Comment (shubheksha) [inline engine/api/get-started.md]: Hey, I think this has been moved too?
https://github.com/moby/moby/blob/master/client/client.go#L136 is accessible

**Comment by shubheksha** [comment]
@mstanleyjones, this could use another review, I've addressed all your comments.
